docs/pages/build/zero-to-one/run-it.mdx

```diff
@@ -36,61 +49,51 @@ You have started the micro-rollup using
 
 ```ts showLineNumbers [index.ts]
 const rollup = await MicroRollup({
-    actionSchemas: [ActionSchema1, ActionSchema2, ...],
-    stateMachines: [machine],
-    ...
+  config: stackrConfig,
+  actionSchemas: [ActionSchema1, ActionSchema2, ...],
+  stateMachines: [machine],
+  ...
 });
 
 await rollup.init() // [!code focus]
 ```
 
 ::::
 
-## Let's Deploy 🚀
+## Let's Run 🏃‍♂ [Sandbox mode]
+
+To be doubly sure that everything is working fine, let's try to run your application and see if it works as expected.
+
+In this example, we will run our micro-rollup in sandbox mode.
 
 :::info
-The below command will package your State Machine and send it to Vulcan for deployment and re-execution on the network.
+**Sandbox mode** 🧪
+
+Sandbox mode doesn't require registration or deployment of your MRU. The SDK creates a temporary SQLite database to store data and the blocks are not sent to Vulcan or L1 for verification.
+It's recommended to use this mode while development and for experimentation.
 :::
```

Comment:
add note on to enable or check if the MRU is in sandbox mode, make sure `isSandbox` in stackrConfig is set to `true`